x-pack/plugins/enterprise_search/public/applications/app_search/icons/cursor_icon.tsx

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+export const CursorIcon: React.FC = ({ ...props }) => (
```

Comment:
Also, if you anticipate us adding this icon to EUI shortly, it would be great to add a TODO here noting that:

```suggestion
// TODO: This icon will be added to EUI soon - we should remove this custom SVG when once it's available to us
export const CursorIcon: React.FC = ({ ...props }) => (
```

Reply:
https://github.com/elastic/kibana/pull/92939/commits/f9fa1ace4cda0fe3e5b669de883ecaadcbc6e3aa